amgut/lib/data_access/ag_data_access.py

```diff
@@ -954,6 +954,29 @@ def handoutCheck(self, username, password):
         else:
             return bcrypt.verify(password, to_check[0])
 
+    def check_access(self, user, barcode):
+        """Check if the user has access to the barcode
+
+        Parameters
+        ----------
+        user : str
+            The user's supplied kit ID
+        barcode : str
+            The barcode to check access for
+
+        Returns
+        -------
+        boolean
+            True if the user can access the barcode, False otherwise
+        """
+        cursor = self.get_cursor()
+        cursor.execute("""SELECT barcode
```

Comment:
why not do something like:
cursor.execute("""SELECT EXISTS SELECT barcode FROM ag.ag_kit JOIN ag.ag_kit_barcodes USING(ag_kit_id) WHERE supplied_kit_id=%s AND barcode=%s""", [user, barcode])
return cursor.fetchone()[0]